### PR TITLE
Model and import area data

### PR DESCRIPTION
**Pull request by struan**

### CODE REVIEW

File: .env-example [hunk 1]
```diff
@@ -1,2 +1,4 @@
 SECRET_KEY="this_IS_secr3t"
 DEVENV="docker"
+MAPIT_URL="https://mapit.mysociety.org/"
```
Comment (zarino) [inline .env-example:3]: Should we also add the `MAPIT_URL` setting to the `web` container’s environment in `docker-compose.yml`?

File: .env-example [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
+MAPIT_API_KEY=""
```
Comment (zarino) [inline .env-example:4]: I wonder what the best way to handles this is, for Docker users. Perhaps `README.md` should be updated, to include an instruction about creating a `.env` file with your MapIt API key in it, before running `docker-compose up`?

File: hub/management/commands/import_areas.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import json
+from django.core.management.base import BaseCommand
+
+from time import sleep
+from utils import mapit
+from hub.models import Area
+
+
+class Command(BaseCommand):
+    help = "Import basic area information from MaPit"
```
Comment (zarino) [inline hub/management/commands/import_areas.py:10]: 10 bonus points for the archaic capitalisation of MapIt. :+1:

I wonder whether `README.md` should explain that you’ll need to run `./manage.py import_areas` after you’ve started the docker container?

Comment (zarino) [reply hub/management/commands/import_areas.py:10]: Might also want to let them know it’ll take at least 10 minutes to run ;-)

File: hub/static/js/all.js [hunk 4]
```diff
@@ -153,18 +155,22 @@ var setUpMap = function() {
...
-                mouseover: highlightFeature,
-                mouseout: unhighlightFeature,
-                click: function(){
-                    window.location.href = '/area'
-                }
-            })
-        }
-    }).addTo(map)
+    $.getJSON("/filter_areas/", function(data) {
+        console.log(data);
```
Comment (zarino) [inline hub/static/js/all.js:159]: Can see why this was left in, but we should probably remove it before we forget!

**Review by zarino — APPROVED** [review]
This all worked for me, once I’d rebuilt the `web` image (to include `requests`) and created an `.env` file with my `MAPIT_API_KEY`.

I’ve left a few comments below, thinking primarily about how we should document setting this stuff up in Docker.

**Review by alexander-griffen — APPROVED** [review]
Runs on my machine well!

**Pull request closed without merging**